### PR TITLE
Notes/GitHub and markdown

### PR DESCRIPTION
**Pull request by manukao**
add files and made changes

### CODE REVIEW

File: session2.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+---
+
+## Notizen
+
+### Shell Basics
+
+- [Handout Shell Basics](../sessions/shell-basics/shell-basics.md)
+
+- `clear` räumt das aktuelle Terminal-Fenster auf, aber alle vergangenen Befehle/Ereignisse sind durch einfaches Hochscrollen wieder sichtbar - die Kommandos sind also nicht gelöscht, sondern nur nach oben verschoben
```
Comment (rabe200) [inline session2.md:36]: extremely important life lesson!

File: session2.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+---
+
+### Git CLI and Remote
+
+- [Handout Git CLI and Remote](../sessions/git-cli-and-remote/git-cli-and-remote.md)
+- `git rm --cached .DS_Store` entfernt die Datei ".DS_Store" aus der git staging area.
+  - `--cached` wird verwendet, um die Datei aus dem Staging-Bereich zu entfernen, ohne sie aus dem Dateisystem zu löschen
+- `git remote set-url origin git@github.com:{USERNAME}/{PROJECTNAME}.git`, um das Repository von HTTPS zu SSH zu wechseln.
+
+  - `git remote` Befehl, um Verbindungen zu anderen Repositorys zum erstellen, anzeigen und löschen.
```
Comment (rabe200) [inline session2.md:53]: don't forget. also important

**Review by rabe200 — COMMENTED** [review]
LGTM

File: session1.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
+### Session 1 Protokol Copy
```
Comment (ella-blaupause) [inline session1.md]: LGTM

File: session1.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+---
+
+date: 06.02.2023, Monday
```
Comment (ella-blaupause) [inline session1.md:5]: cool

**Review by ella-blaupause — COMMENTED** [review]
fine

**Pull request merged**